hazelcast-hibernate53/src/main/java/com/hazelcast/hibernate/local/TimestampsRegionCache.java

```diff
@@ -106,8 +106,4 @@ protected void maybeInvalidate(final Object messageObject) {
             }
         }
     }
-
-    @Override
-    final void cleanup() {
```

Comment:
Can we customize the configuration for timestamps cache or may be make it use relatively higher values than the default ones? Because this no-op override prevents eviction/expiration for timestamps cache and this is also the suggested way according to [Hibernate doc](https://docs.jboss.org/hibernate/orm/5.4/userguide/html_single/Hibernate_User_Guide.html#caching-query-region):
>  In fact, we recommend that the default-update-timestamps-region region is not configured for expiration (time-based) or eviction (size/memory-based) at all.

Reply:
I guess we could simply bring back the old behaviour where 0 meant infinite and not literally 0.